src/main/docker/Dockerfile

```diff
@@ -40,6 +42,24 @@ RUN mkdir -p $TOMCAT_PARENT_DIR && \
 COPY logging.properties .
 RUN cp logging.properties $TOMCAT_CONF_DIR
 
+# Copy new logback.xml and logback-access.xml to tomcat conf folder for logback implementation
+COPY logback.xml .
+RUN cp logback.xml $TOMCAT_CONF_DIR
+
+COPY logback-access.xml .
+RUN cp logback-access.xml $TOMCAT_CONF_DIR
+
+# Copy new setenv.bat and setenv.sh to tomcat bin folder
+COPY setenv.bat .
+RUN cp setenv.bat $TOMCAT_BIN_DIR
+
+COPY setenv.sh .
+RUN cp setenv.sh $TOMCAT_BIN_DIR
```

Comment:
Is this overwriting default files?

Reply:
It won't override setenv.sh and setenv.bat. The tomcat archive don't have these files and these 2 files are new only. 
These are new files added as mentioned in the doc https://github.com/tomcat-slf4j-logback/tomcat9-slf4j-logback

Reply:
In that case just use COPY rather than COPY then RUN.  These 8 commands should be 2 COPY commands.